apps/web/src/api/hooks/notification-templates/useFetchBlueprints.ts

```diff
@@ -1,12 +1,12 @@
 import { useQuery, UseQueryOptions } from '@tanstack/react-query';
-import { IconName } from '@fortawesome/fontawesome-svg-core';
 import localforage from 'localforage';
 import { addWeeks } from 'date-fns';
 import { IGroupedBlueprint } from '@novu/shared';
 
 import { getBlueprintsGroupedByCategory } from '../../notification-templates';
 import { QueryKeys } from '../../query.keys';
 import { IBlueprintTemplate } from '../../types';
+import { getWorkflowBlueprintDetails } from '../../../utils';
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Thank you for extracting this!